storybook/stories/Helpers/style.css

```diff
@@ -0,0 +1 @@
+.col { display: flex; flex-direction: column; }
```

Comment:
lets do this in the same style as the other css, TLDR; run with prettier.

Reply:
I ran prettier and pushed🙂